irohad/model/converters/pb_query_response_factory.hpp

```diff
@@ -21,6 +21,7 @@
 #include <responses.pb.h>
 #include <model/account_asset.hpp>
 #include <model/queries/responses/account_assets_response.hpp>
+#include <model/queries/responses/account_detail_response.hpp>
 #include <model/queries/responses/account_response.hpp>
```

Comment:
Wrong brackets :( should be `"..."`

Reply:
How about other? (Imo, this pr's task is Add get account detail query. so I don't want to update other code)

Reply:
not really huge changes so it's fine